analysis/moa/examples/assertion 1.json

```diff
@@ -0,0 +1,298 @@
+{
+	"ASSERTIONS": {
+		"id": 1, 
+		"context": "Chronic, accelerated, or blast phase", 
+		"description": "The U.S. Food and Drug Administration (FDA) granted approval for bosutinib for adult patients with chronic, accelerated, or blast phase Philidelphia chromosome-positive (Ph+) CML with resistance or intolernace to prior therapy.", 
+		"disease": {
+			"oncotree_code": "CML", 
+			"oncotree_term": "Chronic Myelogenous Leukemia"
+		}, 
+		"therapy_name": "Bosutinib", 
+		"therapy_type": "Targeted therapy", 
+		"clinical_significance": "sensitivity", 
+		"predictive_implication": "FDA-Approved", 
+		"feature_ids": [1], 
+		"source_ids": [1], 
```

Comment:
Yeah, as @korikuzma pointed out, I've made the changes already, but instead of changing "EVIDENCE" to "SOURCES", I changed to "Source of Evidence" correspondingly. Please refer to https://github.com/cancervariants/metakb/pull/41/commits/c1d8671454fb2032cde55b666dabc4885d46a22b
and 
https://github.com/cancervariants/metakb/pull/41/commits/49bc390b05ffa7313b79eb499dea731ddc41b472.

Please let me know if you think just "SOURCES" is better here.